packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Fix hreflang alternate links [#2269](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2269)
 - PDP / PLP: Add page meta data tags that have been defined in BM [#2232](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2232)
 - Send PWA Kit events to Data Cloud [#318] (https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2229)
+- Remove log in test file [#2323](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2323)
```

Comment:
ummmm, I feel like this is unnecessary to be in the CHANGELOG?

The tests are no longer generated in customer's project since v3, so test changes are internal

Reply:
CI will fail if you put changelog for it :(

Reply:
I've just added the `skip changelog` for this PR

Reply:
reverted changelog